### ISSUE TITLE
Thread like macros confuse the linter

### ISSUE THREAD

**Issue by viebel** [borkdude/clj-kondo#347]
**version**

clj-kondo v2019.07.05-alpha

**platform**

Macos

**problem**

```clojure
(mq/with-collection db coll
        (mq/find where)
        (mq/sort sort))
```

clj-kondo complains that `mq/find` is called with 1 arg but expects 2.

mq/with-collection is a thread like macro that passes an additional argument.

**Comment by borkdude** [comment]
@viebel For this you'll need to turn off the arity linter inside the macro.

See: https://github.com/borkdude/clj-kondo/blob/master/doc/config.md#exclude-arity-linting-inside-a-specific-macro-call

**Comment by viebel** [comment]
Awesome thanks
